rllib/models/tf/layers/noisy_layer.py

```diff
@@ -12,7 +12,7 @@
 
 
 class NoisyLayer(tf.keras.layers.Layer if tf else object):
-    """A Layer that adds learnable Noise to some previous layer's outputs.
+    r"""A Layer that adds learnable Noise to some previous layer's outputs.
```

Comment:
Are these docstrings rendered somewhere in the docs? (I couldn't find it).

I'm wondering if we have to remove the double backslashes below

Reply:
Oh, good point - let me try to build the docs and see about this, but I think you're right about double backslashes. I don't think warnings should be generated about bad escape sequences as long as double backslashes are used, so I'll go back and check to make sure about this.

Reply:
Thanks! I'd just like to make sure that, if rendered, the formulas get rendered correctly.